charts/cloudzero-agent/templates/_helpers.tpl

```diff
@@ -30,6 +30,17 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-validator-configuration" .Release.Name -}}
 {{- end}}
 
+{{/*
+  This helper function trims whitespace and newlines from a given string.
+*/}}
+{{- define "cleanString" -}}
```

Comment:
nit, but I believe the best practice is to prefix the name of the named template with the name of the chart, since the named templates are defined globally amongst charts and subcharts.
so, maybe `cloudzero-agent.cleanString` would be good

Reply:
No problem, will scope it 👍

Reply:
done!